workflow/snakemake_rules/chores.smk

```diff
@@ -28,4 +29,7 @@ rule update_example_data:
             --subsample-seed 0 \
             --output-metadata {output.metadata} \
             --output-sequences {output.sequences}
+        cp {output.metadata} /tmp/metadata-pipe
+        tsv-uniq /tmp/metadata-pipe > {output.metadata}
+        rm /tmp/metadata-pipe
```

Comment:
Is there a need for `cp`?

```suggestion
            --output-metadata /tmp/metadata-pipe \
            --output-sequences {output.sequences}
        tsv-uniq /tmp/metadata-pipe > {output.metadata}
        rm /tmp/metadata-pipe
```

Reply:
Right, I was trying to do something more complicated and then forgot there was an easy way. You're right! You can push a fix straight to master or we wait until the duplicate bug is fixed upstream